packages/@uppy/core/src/Uppy.js

```diff
@@ -154,6 +123,9 @@ class Uppy {
       recoveredState: null,
     })
 
+    const getOpts = () => this.opts
+    this.#restricter = new Restricter(getOpts.bind(this), this.i18n)
```

Comment:
```suggestion
    this.#restricter = new Restricter(() => this.opts, this.i18n)
```
but wouldn't it be easier to do?
```suggestion
    this.#restricter = new Restricter(this)
```

Reply:
I don't like this, it should be explicit. I think what I did is fine.

Reply:
Well I think my first suggestion is better and still explicit 😅 Did you miss it?
```suggestion
    this.#restricter = new Restricter(() => this.opts, this.i18n)
```

Reply:
I tried that and it and it doesn't work.